tests/borders/border_plugin.test.ts

```diff
@@ -755,37 +740,31 @@ describe("Grid manipulation", () => {
     expect(model.exportData().borders).toEqual({});
   });
 
-  test("Adding a border on a cell removes it on the adjacent cells if it differs", () => {
-    const model = new Model();
-    const b = DEFAULT_BORDER_DESC;
-    setZoneBorders(model, { position: "bottom", color: "red", style: "dashed" }, ["B2"]);
-    setZoneBorders(model, { position: "right", color: "red", style: "dashed" }, ["A3"]);
-    setZoneBorders(model, { position: "left", color: "red", style: "dashed" }, ["C3"]);
-    setZoneBorders(model, { position: "top" }, ["D2"]);
-    setZoneBorders(model, { position: "external" }, ["B3"]);
-    expect(getBorder(model, "B3")).toEqual({ top: b, bottom: b, left: b, right: b });
-    // deleted as the borders are different
-    expect(getBorder(model, "B2")).toBeNull();
-    expect(getBorder(model, "A3")).toBeNull();
-    expect(getBorder(model, "C3")).toBeNull();
-    // untouched as the border are the same
-    expect(getBorder(model, "D2")).toEqual({ top: DEFAULT_BORDER_DESC });
-  });
-
   test("Setting a *clear* border on a cell removes the adjacent border cell", () => {
     const model = new Model();
     const b = DEFAULT_BORDER_DESC;
     setZoneBorders(model, { position: "all" }, ["A1:C3"]);
     setZoneBorders(model, { position: "clear" }, ["B2"]);
-    expect(getBorder(model, "A1")).toEqual({ top: b, bottom: b, left: b, right: b });
+
+    /**
+     * ----------------     ----------------
+     * | A1 | B1 | C1 |     | A1 | B1 | C1 |
+     * ----------------     ------    ------
+     * | A2 | B2 | C2 |  => | A2   B2   C2 |
+     * ----------------     ------    ------
+     * | A3 | B3 | C3 |     | A3 | B3 | C3 |
+     * ----------------     ----------------
+     */
+
+    expect(getBorder(model, "A1")).toEqual({ top: b, left: b, right: b });
```

Comment:
As discussed, should be top, left, right and bottom. This use case should create 9 zones as they are all differents